avm/res/compute/virtual-machine/main.bicep

```
@@ -133,7 +133,7 @@ param extensionDomainJoinConfig object = {
   enabled: false
 }
 
-@description('Optional. The configuration for the [AAD Join] extension. Must at least contain the ["enabled": true] property to be executed.')
+@description('Optional. The configuration for the [AAD Join] extension. Must at least contain the ["enabled": true] property to be executed. To enroll in Intune, add the setting mdmId: "0000000a-0000-0000-c000-000000000000".')
```

```suggestion
@description('Optional. The configuration for the [AAD Join] extension. Must at least contain the ["enabled": true] property to be executed. To enroll in Intune, add the setting mdmId: "<GUID>".')
```

Does it have to be that explicit ID? If not we could keep it generic. OR - if it must be the `00000` we should, I guess remove the letters from the ID, right?
```suggestion
@description('Optional. The configuration for the [AAD Join] extension. Must at least contain the ["enabled": true] property to be executed. To enroll in Intune, add the setting mdmId: "00000000-0000-0000-0000-000000000000".')
```

---

It has to be the exact sated string